common/config/rush/common-versions.json

```diff
@@ -53,6 +53,12 @@
     "@azure/event-hubs": ["^2.1.4"],
 
     // Allow storage-blob-changefeed and storage-file-datalake to use the preview version of storage-blob.
-    "@azure/storage-blob": ["^12.2.0-preview.1"]
+    "@azure/storage-blob": ["^12.2.0-preview.1"],
+
+    // Allow latest node types for Monitor due to TLS requirements
+    "@types/node": ["^14.0.23"],
```

Comment:
@jeremymeng weren't you able to do something clever to support multiple node versions in the past with core stuff?

Reply:
> @jeremymeng weren't you able to do something clever to support multiple node versions in the past with core stuff?

No, I don't recall this. But as Mike mentioned there were a discussion that is still open.